parameter_sync.py

```diff
@@ -29,13 +31,14 @@ def process_parameters_with_prefix(param_prefix, cred_path, aws_region, aws_acce
     def get_parameters(parameter_names_list):
         parameter_list = []
         if parameter_names_list:
-            result = ssm.get_parameters(Names=parameter_names_list, WithDecryption=True)
-            if result:
-                if 'ResponseMetadata' in result:
-                    if 'HTTPStatusCode' in result['ResponseMetadata']:
-                        if result['ResponseMetadata']['HTTPStatusCode'] == 200:
-                            if 'Parameters' in result:
-                                parameter_list = result['Parameters']
+            for parameter_name in parameter_names_list:
+                result = ssm.get_parameters(Names=[parameter_name], WithDecryption=True)
```

Comment:
If you're only going to get one at a time, why not use get_parameter instead of get_parameters?

For the record, this seems to be WAY less efficient than sending a list of parameter names. If there is a limit to how many parameters you can put in the list (not documented, but clearly a problem) then why not break up the list in chunks of size X and call this method multiple times?

Reply:
I've changed it to get_parameter.
This is much less efficient, but the get_paramter call is very quick.